SimG4Core/PhysicsLists/src/CMSEmStandardPhysicsLPM.cc

```diff
@@ -271,6 +274,9 @@ void CMSEmStandardPhysicsLPM::ConstructProcess()
                particleName == "xi_c+" ||
                particleName == "xi-" ) {
 
+      if(nullptr == hmsc) {
+	hmsc = new G4hMultipleScattering("ionmsc");
```

Comment:
@davidlange6 , all classes instantiated inside this method are controlled by a process store. Old code was working properly, because it is guaranteed that the class is created and will be deleted but the static analyser does not know about. So, the fix is a cosmetic one  to remove false positive.